### PR TITLE
Redesign

### PR DESCRIPTION
**Pull request by tnmy44**
List of changes are:
    * Redesign of front end
    * Implementation of announcement system
    * Fixing bug which could cause the app to break when specific data is sent to it by the client
    * Changed avatars

### CODE REVIEW

**Comment by captn3m0** [comment]
Single commit, really?

**Comment by tnmy44** [comment]
There was so much trash in those commits that I had to squash them all into
a single one.

File: announcements.coffee [hunk 1]
```diff
@@ -0,0 +1,48 @@
+
+# ioObject is the express().http().io().io object for communicating with clients
+module.exports = (ioObject, slackObject) ->
```
Comment (apsdehal) [inline announcements.coffee]: This whole file looks a lot close to JavaScript rather than coffeescript.

Comment (tnmy44) [inline announcements.coffee]: I'l look into coffeescript coding conventions and refactor it.

File: public/x-user-list.html [hunk 2]
```diff
@@ -7,13 +7,45 @@
...
+      .username{
+        display: inline-block;
+        margin-top: 8px;
+        font-size: 13px;
+        font-weight: 400;
+      }
+      .list-item{
+        min-height: 40px;
+        display:block;
+        white-space: nowrap;
```
Comment (apsdehal) [inline public/x-user-list.html]: In the separate file please.

File: public/style.css [hunk 3]
```diff
@@ -1,109 +1,117 @@
...
-.avatar.navy {
-    border-color: #393b79;
+#right-panel{
+     padding: 20px 25px 20px 30px;
+     overflow-y: scroll;
 }
-.avatar.slate {
-    border-color: #6b6ecf;
+
+#announcement-title{
```
Comment (apsdehal) [inline public/style.css]: Don't use ids for css styles descriptions, always use classes.

File: public/x-user-list.html (hunk 2 above)
Comment (tnmy44) [inline public/x-user-list.html]: Done.

**Pull request closed without merging**